core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.ts

```diff
@@ -418,18 +418,19 @@ export class TranslationSuggestionReviewModalComponent implements OnInit {
 
   acceptAndReviewNext(): void {
     this.finalCommitMessage = this.generateCommitMessage();
-    if (this.translationUpdated) {
-      this.reviewMessage = this.reviewMessage + ': This suggestion' +
-        ' was submitted with reviewer edits.';
-    }
+    const reviewMessageForSubmitter = this.reviewMessage + (
+      this.translationUpdated ? (
+        (this.reviewMessage.length > 0 ? ': ' : '') +
+        'This suggestion was submitted with reviewer edits.') :
```

Comment:
Should add the "(Note ...)" suggestion as mentioned in https://github.com/oppia/oppia/issues/17936#issuecomment-1501489691 and update the tests.